RELEASE.md

```diff
@@ -8,12 +8,12 @@ Publishing the container is automated through gitlab-ci and only requires one to
 Publishing the helm chart is currently manual, and we should move to an automated process ASAP
 
 # Release Process Checklist
+- [ ] Run the `./hack/prepare-release.sh` script to update the version in all the needed files and generate the Helm chart
```

Comment:
This doesn't generate the helm charts in its current iteration? Should we call the relevant script at the end of `prepare-release.sh`?

Reply:
yup